pycbc/population/live_pastro_utils.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from . import live_pastro as livepa
+
+
+def insert_live_pastro_option_group(parser):
+    """ Add low-latency p astro options to the argparser object.
+
+    Parameters
+    ----------
+    parser : object
+        ArgumentParser instance.
+
+    Returns
+    -------
+    live_pastro_group :
+        Argument group object
+    """
+
+    live_pastro_group = parser.add_argument_group('Options for live p_astro '
+                                                  'calculation')
+
+    # Only one choice so far, allow for more later
+    live_pastro_group.add_argument('--p-astro-method',
+                                   choices=['template_param_bins'])
+    live_pastro_group.add_argument('--p-astro-spec-data',
+        help='File containing information to set up p_astro calculation')
```

Comment:
Have you considered having a single `--p-astro-definition` option taking the JSON file name as argument, and defining the method with an additional field in the JSON file? That seems a bit more compact to me, and it allows us to drop the verify checks below.